react/components/Buttons/ChangeToppings.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import Button from 'vtex.styleguide/Button'
+import { injectIntl, intlShape } from 'react-intl'
+
+class ChangeToppings extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    enableChangeToppings: PropTypes.bool,
+  }
+
+  render() {
+    const {
+      intl,
+      enableChangeToppings,
+    } = this.props
+    const changeLabel = intl.formatMessage({ id: 'product-customizer.change-composition' })
+
+    return (
+      <div className={`actions--change-toppings ph5 pa5 ${!enableChangeToppings ? 'dn' : ''}`}>
```

Comment:
Do not you think that should be better to not render the component when the `enableChangeToppings` is `false` instead of hide the `div`?